Week2/homework/maartjes-work.js

```diff
@@ -43,19 +43,28 @@ const tuesday = [
 ];
 
 const maartjesTasks = monday.concat(tuesday);
+
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
   // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const mappedTasks = tasks.map(task => task.duration / 60);
```

Comment:
The result of mapping the tasks to hours produces an array of hours, not of tasks. So the name `mappedTasks` is incorrect and misleading. A better name would be `hours`.

```js
let hours = tasks.map(task => task.duration / 60);
```